app/views/volunteers/_form.html.slim

```diff
@@ -64,8 +64,8 @@ fieldset
   .col-xs-12
     = f.input :motivation
   .col-xs-12
-    = f.input :experience
-  .col-xs-12
+    = checkbox_toggle_collapse(f, :experience, '#experience-desc')
```

Comment:
Nice change, not part of the story though.

Reply:
I know, but as the test for this was failing I had to fix it. And I'd rather fix it right instead of keeping it as wrecked. So if we consider fixing tests as part of the story, it is... 😉